### DISCUSSION TITLE
vc-drawing-polyline

### DISCUSSION THREAD

**Discussion by xiaowunimte** [zouyaoji/vue-cesium#622] [Q&A]
![image](https://github.com/user-attachments/assets/37959ee2-30a8-45c5-b4dc-dfd6ecc27791)
绘制折线的点如何垂直地面向下打射线(或者不通过鼠标事件如何获取这些点的实例)

**Comment by zouyaoji** [comment]
通过ref模板引用拿到绘制的renderDatas就可以拿到
